### PR TITLE
The requirements for the Tailscale service are not described #471

### LINKED ISSUE #471: The requirements for the Tailscale service are not described
We currently do not list the requirements for the configuration of the Tailscale service:
- install the Tailscale repo 
- install the `tailscale` package

This might have already been needed by a user of the Testing channel: [https://forum.rockstor.com/t/unknown-internal-error-doing-a-post-to-api-sm-services-tailscaled-config/9418/2](https://forum.rockstor.com/t/unknown-internal-error-doing-a-post-to-api-sm-services-tailscaled-config/9418/2?u=flox).

We should provide a link to the Tailscale docs that provide clear step-by-step instructions: https://tailscale.com/download/linux/opensuse-leap-15.2

Note that this page has two methods:
- a pipe to bash script: this will handle up to Leap 15.5 and tumbleweed 
- manual install: this has a drop-down list of supported distros that unfortunately list only Leap 15.1 and 15.2 with tumbleweed. This is unfortunately outdated as the repos currently exist up to 15.5. Note that someone already reported this and the issue is marked as completed https://github.com/tailscale/tailscale/issues/9404).

---

Partner doc issue: https://github.com/rockstor/rockstor-core/issues/2845

### PR DESCRIPTION
**Pull request by phillxnet**
Provide a new light-weight upstream instructions wrapper How-to, linked from our existing Web-UI Tailscale `service` section. And, in-turn, the How-to links back to the service section for configuration and enablement.

Fixes #471

### Checklist
- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

**Comment by phillxnet** [comment]
@FroggyFlox Give this changes a tad your original docs contribution, and that you are more familiar with our Tailscale integration, given you coded it in the first place, could you take a quick look at this before we merge. I think it basically covers our first pass at this doc enhancement, and gets folks off-the-ground on this new existing feature. And hopefully keeps thinks simple regarding our stance re NOT pre-installing this cloud-based capability.

Plus it serves existing installs re the How-to link from our Tailscale services section and we can, once published, consider liking folks directly the 'Install Tailscale' How-to provided contextually upon no-install state such as is proposed in:

https://github.com/rockstor/rockstor-core/issues/2845 Which should probably be re-named now given our new stance re NOT pre-installing and not offering an automated install (at least yet).

We can always re-visit the added How-to re an OBS alternative repo section etc: but upstream is often more the go-to on such things anyway. So we should at least start with the vanilla, and likely most supported option anyway.

I'm not keen on any install-via-curl advise so have mentioned only the zypper approach.

File: howtos/tailscale_install.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Upstream instructions
+---------------------
+
+This How-to is derived from, and secondary to,
+the canonical upstream instructions as provided by `Tailscale <https://tailscale.com/>`_ themselves.
+It is intended only to assist in providing relevant links and light guidance.
+
+.. note::
+
+    Rockstor is "Build on openSUSE", specifically **openSUSE Leap** or **Tumbleweed**: depending on the installer used.
```
Comment (FroggyFlox) [inline howtos/tailscale_install.rst:24]: Minor typo:
```suggestion
    Rockstor is "Built on openSUSE", specifically **openSUSE Leap** or **Tumbleweed**: depending on the installer used.
```

**Review by FroggyFlox — APPROVED** [review]
@phillxnet 
Thank you so much for taking care of yet another issue.
This looks great!

File: howtos/tailscale_install.rst (hunk 1 above)
Comment (phillxnet) [reply howtos/tailscale_install.rst]: @FroggyFlox Thanks, and much appreciated. I'm always surprised at my missing these. I'd already corrected identical typos before submission also; and it's only a small How-to!

**Comment by phillxnet** [comment]
PR product PRODUCTION published.

**Pull request merged**